src/Serialize.h

```diff
@@ -33,6 +33,101 @@ namespace drafter {
 
     refract::Registry& GetNamedTypesRegistry();
 
+    template<typename T>
+    struct SectionInfo {
+        typedef SectionInfo<T> Type;
+        typedef T SectionType;
+        typedef snowcrash::SourceMap<SectionType> SourceMapType;
+
+        const SectionType& section;
```

Comment:
> There is a struct ParseResultRef just for that in snowcrash. As I start looking at the code, I am more in favour of using ParseResult related stuff.

In drafter we are working with "const reference". Do you mean we should create another structure"ParseResultConstRef"?